pkg/controller/certificates/certificate_controller.go

```diff
@@ -181,3 +190,16 @@ func (cc *CertificateController) syncFunc(key string) error {
 
 	return cc.handler(csr)
 }
+
+// IgnorableError returns an error that we shouldn't handle (i.e. log) because
```

Comment:
Maybe clean up this comment to explain why we need `error`s that aren't actually errors.